### ISSUE TITLE
SR Flip-Flop doesn't have clock 

### ISSUE THREAD

**Issue by Prerna-0202** [CircuitVerse/CircuitVerse#3032]
**Describe the bug**
In the Simulator, the SR Flip Flop does not have clock, due to which SR Flip Flop is showing some unexpected behavior. 

**To Reproduce**
1. Go to 'Simulator'
2. Scroll down to 'Sequential Elements'
3. Click On 'SR Flip-Flop'
4. See error.

**Expected behavior**
It should have a clock(like JK Flip Flop).

**Screenshots**
Unexpected behavior : 

https://user-images.githubusercontent.com/89515816/158028014-90ed3df8-3f73-4a7b-a39a-8566046dd43f.mov

**Device Information [optional]:**
 - OS: Windows
 - Browser :Chrome

**Additional context**

**Comment by khareyash05** [comment]
I would like to work on this if this issue is open

**Comment by Prerna-0202** [comment]
> I would like to work on this if this issue is open

Yes you can , but let just wait for  @gr455 and @tachyons suggestions on this issue..
